### DISCUSSION TITLE
Can Prism.js do code folding?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [PrismJS/prism#3807] [Q&A] [answered]
![image](https://github.com/PrismJS/prism/assets/12727582/2a4fab4b-bda1-466b-8772-8c2413f779f8)

Prism.js can highlight code and display line numbers: 
Is it possible to fold the code?

**Comment by LadyBeGood** [comment, marked as answer]
Yes, it's possible for JavaScript and JSON using this plugin: [prism-js-fold](https://github.com/nlundquist/prism-js-fold).

For other languages, I couldn't find Prism-specific solutions. As an alternative, consider using an embedded code editor like [Ace](https://ace.c9.io/) which has built-in folding support for many languages.
